autoload/flagship.vim

```diff
@@ -520,9 +520,16 @@ function! flagship#flags_for(type) abort
   let flags = []
   for [F, opts; rest] in exists('s:flags') ? get(s:flags, a:type, []) : []
     let str = join([F])
+    let is_function = type(F) == type(function('tr'))
     unlet! F Hl
     if !empty(get(g:, 'flagship_skip', '')) && str =~# g:flagship_skip
       let flag = ''
+    elseif is_function
+      if str =~# '^function('
+        let flag = '%{flagship#call('.str.')}'
+      else
+        let flag = '%{flagship#call(function('.string(str).'))}'
+      endif
```

Comment:
I think all this can collapse down to `elseif str =~# '^function('`, no? It's already carefully constructed to accommodate named functions in the same way as strings.

Reply:
The `luaeval` return is `<lambda>425` rather than `function('<lambda>425')`, but it can be done on string matching.  I'll push up a commit with that change.